core/src/pixelcolor/mod.rs

```diff
@@ -23,10 +24,33 @@
 //!     Red,
 //! }
 //!
-//! /// The `Raw` can be is set to `()` because `EpdColor` doesn't need to be
-//! /// converted to raw data for the display and isn't stored in images.
+//! /// Implement the `PixelColor` trait to mark this type as a embedded-graphics
+//! /// color and to specify the binary storage format.
```

Comment:
I had a go at wording this a little better:

```suggestion
//! /// Implement the `PixelColor` trait to mark this type as an embedded-graphics
//! /// color and associated binary storage format.
```

Reply:
Thanks, that's much better.